src/realm/group.hpp

```diff
@@ -1002,7 +1002,7 @@ inline ref_type Group::get_history_ref(const Array& top) noexcept
 inline int Group::get_history_schema_version(const Array& top) noexcept
 {
     if (top.is_attached()) {
-        if (top.size() >= 8) {
+        if (top.size() >= 9) {
```

Comment:
Note that this commit reverts this change, i.e., it uses `>= 8` rather than `>= 9`.